HLTrigger/Muon/plugins/HLTMuonL1TFilter.cc

```diff
@@ -110,7 +118,14 @@ bool HLTMuonL1TFilter::hltFilter(edm::Event& iEvent,
       MuonRef muon(allMuons, distance(allMuons->begin(allMuons->getFirstBX()), it));
 
       // Only select muons that were selected in the previous level
-      if (find(prevMuons.begin(), prevMuons.end(), muon) == prevMuons.end())
+      bool matchPrevL1 = false;
+      int prevSize = prevMuons.size();
+      for (int it2 = 0; it2 < prevSize; it2++) {
+        if (deltaR2(muon->eta(), muon->phi(), prevMuons[it2]->eta(), prevMuons[it2]->phi()) < maxDR2_)
+          matchPrevL1 = true;
+        break;
```

Comment:
```suggestion
        if (deltaR2(muon->eta(), muon->phi(), prevMuons[it2]->eta(), prevMuons[it2]->phi()) < maxDR2_) {
          matchPrevL1 = true;
          break;
        }
```
Careful with brackets :)

Reply:
Yikes, I am really not awake enough today to code. Fix incoming...

Reply:
@JanFSchulte , while you are at it, consider squashing into 1 commit (optional).

Reply:
Sure, done.